frontend/src/redux/auth/actions.js

```diff
@@ -7,27 +7,66 @@ export const authLoginAction = (userToken) => {
   };
 };
 
+// SIGNUP
 export const authSignupStartAction = () => {
   return {
     type: actions.AUTH_SIGNUP_START,
   };
 };
-
 export const authSignupSuccessAction = (message, token) => {
   return {
     type: actions.AUTH_SIGNUP_SUCCESS,
     message,
     token,
   };
 };
-
 export const authSignupFailAction = (message) => {
   return {
     type: actions.AUTH_SIGNUP_FAIL,
     message,
   };
 };
 
+// VALIDATION
+export const authCodeValidationStartAction = () => {
+  return {
+    type: actions.AUTH_VALIDATION_START,
+  };
+};
+export const authCodeValidationSuccessAction = (message, token) => {
+  return {
+    type: actions.AUTH_VALIDATION_SUCCESS,
+    message,
+    token,
+  };
+};
+export const authCodeValidationFailAction = (message) => {
+  return {
+    type: actions.AUTH_VALIDATION_FAIL,
+    message,
+  };
+};
+
+// INFO UPDATE
+export const authInfoUpdateStartAction = () => {
+  return {
+    type: actions.AUTH_INFOUPDATE_START,
+  };
+};
+export const authInfoUpdateSuccessAction = (message, token) => {
+  return {
+    type: actions.AUTH_INFOUPDATE_SUCCESS,
+    message,
+    token,
```

Comment:
I'm not sure if that request sends a token again (since we receive the token in the first signup phase already), but even if that request sends us a token you don't use that token parameter in the infoupdatesuccessReducer so, it makes no problems but it is ,you know, unnecessary.

Reply:
You're right, line 60 is no more.